x-pack/plugins/cloud_security_posture/server/create_indices/ingest_pipelines.ts

```diff
@@ -52,3 +53,24 @@ export const latestFindingsPipelineIngestConfig: IngestPutPipelineRequest = {
     },
   ],
 };
+
+export const latestVulnerabilitiesPipelineIngestConfig: IngestPutPipelineRequest = {
+  id: CSP_LATEST_VULNERABILITIES_INGEST_TIMESTAMP_PIPELINE,
+  description: 'Pipeline for cloudbeat latest vulnerabilities index',
+  processors: [
+    {
+      set: {
+        field: 'event.ingested',
+        value: '{{_ingest.timestamp}}',
+      },
+    },
+  ],
+  on_failure: [
+    {
+      set: {
+        field: 'error.message',
+        value: '{{ _ingest.on_failure_message }}',
+      },
+    },
+  ],
+};
```

Comment:
I am afraid that with `Promise.all` if when one of the methods will be rejected the others will not be completed as well, 
what do you think? 

@JordanSh